lib/modules/showImages.js

```diff
@@ -1303,6 +1300,22 @@ function setMediaClippyText(elem) {
 	elem.setAttribute('title', title);
 }
 
+function setMediaMaxSize(media) {
+	let value = module.options.maxWidth.value;
+	let maxWidth = parseInt(value, 10);
+	if (maxWidth > 0) {
+		if (_.isString(value) && value.match('%')) maxWidth *= document.documentElement.clientWidth / 100;
```

Comment:
`value.endsWith('%')` or `value.includes('%')` instead of `match`

(same below)